src/ngMock/angular-mocks.js

```diff
@@ -1346,11 +1346,15 @@ function createHttpBackendMock($rootScope, $timeout, $delegate, $browser) {
                  copy(response[3] || ''));
       }
 
-      function handleTimeout() {
+      function handleTimeout(statusText) {
         for (var i = 0, ii = responses.length; i < ii; i++) {
           if (responses[i] === handleResponse) {
             responses.splice(i, 1);
-            callback(-1, undefined, '');
+            if (statusText) {
```

Comment:
The advice in the docs is that _"you should never need to use this service directly"_ and it's use as a function isn't even documented (even more so the arguments of the `done` callback).
I really think it's a non-issue, so go ahead and use the simpler version :smile: